src/server/server.ts

```diff
@@ -30,11 +30,9 @@ app.set('trust proxy', 'loopback');
 
 //http to https redirects
 app.enable('trust proxy');
-app.use((req,res,next)=>{
-  if(rec.secure){
-    next();
-  }else{
-    res.redirect('https://'+req.host+req.url);
+app.use(function(request, response, next)=>{
+  if (request.protcol == 'http') {
```

Comment:
protocol

Q: can't we just add listener to the ALB?

Reply:
yes, however AWS does not make it easily to automate and that's why i have decided to try to do it in node.